templates/templates/preview/experiment_html.template

```diff
@@ -0,0 +1,6 @@
+<p>=== Experiment Overview ===</p>
+<ul style="list-style-type:none;">
```

Comment:
Don't do inline styles.
- move style declarations to the base template, set class here to something like `entityList` so we don't have to change inline styles

Reply:
changed